Bottom: 7079bf2fd2ee20cb937df5d1ed91d7f84755f470
Top:    97e3453fea328c4183f886c8a2f090b48c9dc150
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 14:17:10 +0800

Refresh of vhost

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 52fc4d8..bc41721 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1080,7 +1080,6 @@ static long vhost_net_set_backend(struct vhost_net *n, unsigned index, int fd)
 
 		vhost_net_disable_vq(n, vq);
 		vq->private_data = sock;
-		/* FIXME: only for RX virtqueue */
 		nvq->rx_array = get_tap_skb_array(fd);
 		r = vhost_vq_init_access(vq);
 		if (r)
